modules/installation-registry-osp-creating-custom-pvc.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring_registry_storage/configuring-registry-storage.adoc
+
+[id="installation-registry-osp-creating-custom-pvc_{context}"]
+= Configuring an image registry with a custom back end on clusters that run on {rh-openstack}
+
+After you install a cluster on {rh-openstack-first}, you can use a Cinder volume that is in a specific availability zone for registry storage.
+
+.Procedure
+
+. Create a YAML file that specifies a storage class that includes the availability zone that you want to use. For example:
```

Comment:
Does the following lose meaning?

Create a YAML file that specifies the storage class and availability zone to use.

Reply:
```suggestion
. Create a YAML file that specifies the storage class and availability zone to use. For example:
```